cmake/modules/FindSUNDIALS.cmake

```diff
@@ -58,48 +60,42 @@ DEAL_II_FIND_LIBRARY(SUNDIALS_LIB_SER NAMES sundials_nvecserial
   PATH_SUFFIXES lib${LIB_SUFFIX} lib64 lib
   )
 
-#
-# only define the extra hint if ${SUNDIALS_DIR} is nonempty so that we do not
-# try to search through the top-level '/include/' directory, should it happen to
-# exist
-#
-STRING(COMPARE EQUAL "${SUNDIALS_DIR}" "" _sundials_dir_is_empty)
-IF(NOT ${_sundials_dir_is_empty})
-  SET(_sundials_include_hint_dir "${SUNDIALS_DIR}/include/")
-ENDIF()
-
 DEAL_II_FIND_PATH(SUNDIALS_INCLUDE_DIR sundials/sundials_nvector.h
-  HINTS ${SUNDIALS_DIR} "${_sundials_include_hint_dir}"
+  HINTS ${SUNDIALS_DIR}
+  PATH_SUFFIXES include
```

Comment:
@luca-heltai @drwells What about simply using `PATH_SUFFIXES include` in this instance? :-D

Reply:
It's fine with me, but @drwells had a case where things were not working with just `include`, if I recall correctly.